--- origsrc/ocaml-faad-0.3.0/configure	2011-06-30 14:36:42.000000000 -0500
+++ src/ocaml-faad-0.3.0/configure	2011-12-15 19:36:19.605410100 -0600
@@ -2196,7 +2196,7 @@ fi
 ac_config_headers="$ac_config_headers config.h:config.h.in"
 
 # Include it
-CFLAGS="$CFLAGS -I`pwd`/`dirname $0` -DHAVE_CONFIG_H"
+CFLAGS="$CFLAGS -I`pwd` -DHAVE_CONFIG_H"
 
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
--- origsrc/ocaml-faad-0.3.0/configure.ac	2011-06-30 14:36:42.000000000 -0500
+++ src/ocaml-faad-0.3.0/configure.ac	2011-12-15 19:36:08.439771400 -0600
@@ -16,7 +16,7 @@ fi
 # Include a config.h
 AC_CONFIG_HEADERS([config.h:config.h.in])
 # Include it
-CFLAGS="$CFLAGS -I`pwd`/`dirname $0` -DHAVE_CONFIG_H"
+CFLAGS="$CFLAGS -I`pwd` -DHAVE_CONFIG_H"
 
 AC_CHECK_HEADERS([errno.h])
 
